extensions/amp-form/0.1/test/test-amp-form-textarea.js

```diff
@@ -37,9 +38,18 @@ describes.realWin('amp-form textarea[autoexpand]', {
     sandbox = env.sandbox;
   });
 
-  describe('AmpFormTextarea', () => {
+  describe('handleInitialOverflowElements', () => {
     it('should remove autoexpand on elements with initial overflow', () => {
+      const textarea = doc.createElement('textarea');
```

Comment:
Might be a bit cleaner using `htmlFor` (except for the text).

Reply:
leaving for a followup PR to align with other test cases